.vscode/settings.json

```diff
@@ -2,8 +2,10 @@
   "editor.codeActionsOnSave": {
```

Comment:
Could we actually add the IDE-related files to `.gitignore` and remove them from the repo?